sktime/forecasting/ets.py

```diff
@@ -299,14 +305,20 @@ def _fit(error, trend, seasonal, damped):
                 )
             )
 
+            # Store IC values for each model in a list
+            # Ignore infinite likelihood models if ignore_inf_ic is True
+            _ic_list = []
+            for result in _fitted_results:
+                ic = getattr(result[1], self.information_criterion)
+                if ic == -np.inf and self.ignore_inf_ic:
```

Comment:
```suggestion
                if self.ignore_inf_ic and np.isinf(np.inf):
```

Reply:
Thanks @mloning! np.isinf is more elegant indeed.